pkg/labels/selector.go

```diff
@@ -301,6 +307,8 @@ func (lsel internalSelector) Matches(l Labels) bool {
 	return true
 }
 
+func (lsel internalSelector) Requirements() ([]Requirement, bool) { return lsel, true }
```

Comment:
Somewhere, name the []Requirement type and document that it's an implicit AND of all the elements?